src/game/features/players/toxic/LightningStrike.cpp

```diff
@@ -0,0 +1,22 @@
+#include "game/commands/PlayerCommand.hpp"
+#include "game/features/Features.hpp"
+#include "game/pointers/Pointers.hpp"
+#include "game/rdr/Enums.hpp"
+#include "game/rdr/Natives.hpp"
+
+
+namespace YimMenu::Features
+{
+	class LightningStrike : public PlayerCommand
+	{
+		using PlayerCommand::PlayerCommand;
+
+		virtual void OnCall(Player player) override
+		{
+			auto playerCoords = player.GetPed().GetPosition();
+			FIRE::ADD_EXPLOSION(playerCoords.x, playerCoords.y, playerCoords.z, 33, 5.0f, true, false, 5.0f);
```

Comment:
I think there is an explosion type enum that can be used here

Reply:
ExplosionTypes:: SPARKS_WITH_FIRE_AFTERWARDS might be what is best, i have not tested though

Reply:
Altered, thanks!  i wasnt sure which one to use lol

Reply:
> ExplosionTypes:: SPARKS_WITH_FIRE_AFTERWARDS might be what is best, i have not tested though

That explosion type is not the type for lightning sparks, i did test that and it was not the same explosion type so this was reverted @Rxann